### PR TITLE
openapi: document operations for admin ui feedback

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR updates the endpoint documentation and schemas related to Admin UI feedback.

### CODE REVIEW

**Review by kwasniew — APPROVED** [review]

File: src/lib/routes/admin-api/user-feedback.ts [hunk 1]
```diff
@@ -60,8 +67,14 @@ class UserFeedbackController extends Controller {
...
                     tags: ['Admin UI'],
                     operationId: 'updateFeedback',
+                    summary: 'Update Unleash feedback',
+                    description:
+                        'Updates the feedback with the provided ID. Only provided fields are updated. Fields left out are left untouched.',
                     requestBody: createRequestSchema('feedbackSchema'),
-                    responses: { 200: createResponseSchema('feedbackSchema') },
+                    responses: {
+                        200: createResponseSchema('feedbackSchema'),
+                        ...getStandardResponses(400, 401, 415),
```
Comment (gastonfournier) [inline src/lib/routes/admin-api/user-feedback.ts:76]: Can we get a 404 if the id does not exist?

Comment (thomasheartman) [reply src/lib/routes/admin-api/user-feedback.ts:75]: Good question! Let me find out!

Comment (thomasheartman) [reply src/lib/routes/admin-api/user-feedback.ts:75]: You should! Right now you might get a 500 😳

Comment (gastonfournier) [reply src/lib/routes/admin-api/user-feedback.ts:75]: I found similar situations on other update endpoints!

File: src/test/e2e/api/admin/feedback.e2e.test.ts [hunk 2]
```diff
@@ -33,7 +33,13 @@ beforeAll(async () => {
         );
     };
 
-    app = await setupAppWithCustomAuth(stores, preHook);
+    app = await setupAppWithCustomAuth(stores, preHook, {
+        experimental: {
+            flags: {
+                strictSchemaValidation: true,
```
Comment (gastonfournier) [inline src/test/e2e/api/admin/feedback.e2e.test.ts:39]: Nice!

**Review by gastonfournier — APPROVED** [review]
Looks good! Maybe we can use this opportunity to improve the NotFound returning a 500, but can also be part of a different PR

File: src/lib/routes/admin-api/user-feedback.ts (hunk 1 above)
Comment (kwasniew) [reply src/lib/routes/admin-api/user-feedback.ts:75]: Same here. I try to turn those 5xx into client 4xx errors whenever I test some endpoint. As as general rule of thumb if the endpoint has id or some other query/path param we may need to test for 400 vs 500 handling

Comment (thomasheartman) [reply src/lib/routes/admin-api/user-feedback.ts:75]: No, it does not. If it doesn't exist, we create that feedback id..

**Comment by thomasheartman** [comment]
@gastonfournier The 500 wasn't because the ID was not found, but because I used an admin token. You need an identifiable user to post and put these.

File: src/lib/routes/admin-api/user-feedback.ts (hunk 1 above)
Comment (thomasheartman) [reply src/lib/routes/admin-api/user-feedback.ts:75]: Update: this only returns 500 if you use an admin token (and thus have no identifiable user). @kwasniew , I think we might have discussed cases like this before? I don't know if it's worth adding the handling for this in this one specific place now, but I can definitely add it to the description.

Comment (kwasniew) [reply src/lib/routes/admin-api/user-feedback.ts:75]: I remember us having this conversation but I don't remember how we solved it. @sjaanus do you remember what we did for the admin tokens that had no user id?

Comment (sjaanus) [reply src/lib/routes/admin-api/user-feedback.ts:75]: I think we never solved it, but the consensus was that are moving away from admin tokens and endpoints that require user as foreign key, can not work for admin tokens.

Comment (thomasheartman) [reply src/lib/routes/admin-api/user-feedback.ts:75]: In that case, I'm going to merge it for now and we can update that later if we want to.

**Pull request merged**